composer.json

```diff
@@ -1,6 +1,6 @@
 {
-    "name": "chobie/jira-api-restclient",
-    "description": "JIRA REST API.",
+    "name": "t-keller/jira-api-restclient",
```

Comment:
To use your fork instead of original package you need to specify this in your `composer.json` file:

```json
"repositories": [
        {
            "type": "vcs",
            "url": "https://github.com/t-keller/jira-api-restclient"
        }
    ],
```